Mendelugram/Mendelugram/View/PhotoDescription/PhotoDescriptionTableViewCell.swift

```diff
@@ -7,25 +7,24 @@
 
 import UIKit
 
-class PhotoDescriptionTableViewCell: UITableViewCell {
-
-    static let reuseIdentifier = String(describing: PhotoDescriptionTableViewCell.self)
+class PhotoDescriptionTableViewCell: UITableViewCell, NibNameIdentifiable {
 
     struct Input {
         let likesCount: Int
         let username: String
         let description: String
     }
 
-    @IBOutlet weak var likesCountLabel: UILabel!
-    @IBOutlet weak var descriptionLabel: UILabel!
-
-    override func awakeFromNib() {
-        super.awakeFromNib()
-    }
+    @IBOutlet private weak var likesCountLabel: UILabel!
+    @IBOutlet private weak var descriptionLabel: UILabel!
 
-    func configure(input: Input) {
-        likesCountLabel.text = "\(input.likesCount) To se mi líbí"
+    func configure(with input: Input) {
+        let formatter = NumberFormatter()
```

Comment:
Initializing the same formatter over and over again

Reply:
It's better to init it every time IMHO then store a reference and waste memory.

Reply:
also it's more readable

Reply:
I rather waste memory than time but fair enough.